contracts/enable-kv/README.md

```diff
@@ -0,0 +1,21 @@
+## Summary
+This README illustrastes the steps for usage of enable-kv.sh script with kv_map as an example. Set your environment variables as follows before you begin the rest of the steps.
+1. export EOS_3_0=[eos 3.0 directory]
+1. export EOSIO_CDT_3_0=[eosio.cdt 3.0 directory]
+1. export PATH=$EOS_3_0/build/bin:$PATH
+
+## Steps
+Bring up nodeos in a different terminal
+1. export PATH=$EOS_3_0/build/bin:$PATH
+1. nodeos -e -p eosio --plugin eosio::producer_plugin --plugin eosio::producer_api_plugin --plugin eosio::chain_api_plugin --plugin eosio::http_plugin --plugin eosio::history_plugin --plugin eosio::history_api_plugin --filter-on=* --access-control-allow-origin=* --contracts-console --http-validate-host=false --verbose-http-errors --max-transaction-time=1000 --backing-store chainbase --data-dir=datadir
+
+In the first terminal
+1. $EOS_3_0/contracts/enable-kv/enable-kv.sh -c $EOS_3_0/build/contracts/contracts/
+1. cd $EOSIO_CDT_3_0/examples/kv_map
+1. mkdir build
+1. cd build
+1. cmake .. -DCMAKE_PREFIX_PATH=$EOS_3_0/build -DEOSIO_CONTRACTS_ROOT=$EOSIO_CDT_3_0/build/contracts
```

Comment:
$EOSIO_CDT_3_0/build does not have contracts directory. Set another environment for eos-contracts. 

Have you run through the steps?

Reply:
The assumption is that eos is built in $EOSIO_CDT_3_0/build/ (which is the standard practice). We can follow the standard practice, or build $EOSIO_CDT_3_0/contracts/contracts separately in a different directory, and create another environment variable. 

What's the preference?

Reply:
In your '-DEOSIO_CONTRACTS_ROOT=$EOSIO_CDT_3_0/build/contracts' option, there is no such directory '$EOSIO_CDT_3_0/build/contracts'.

Reply:
Apparently, cmake skipped processing -DEOSIO_CONTRACTS_ROOT (as the directory was not found), and rest of the steps went through fine. 

Not sure, if it is even required. Can you please confirm?

Reply:
Please take it and retest. If all steps pass, we are good.

Reply:
Removed unnecessary cmake command line option.